### PR TITLE
Use bulkCreate instead of create loop on subscription.

### PR DESCRIPTION
**Pull request by jnaviask**
Needs QA, particularly email refactor.

### CODE REVIEW

File: server/models/subscription.ts [hunk 1]
```diff
@@ -206,9 +213,11 @@ export default (
           notification.id
         }`;
       }
-      if (msg && subscription.immediate_email) sendImmediateNotificationEmail(subscription, msg);
-      return nRead;
-    }));
+      if (msg && nRead.Subscription.immediate_email) {
```
Comment (zakhap) [inline server/models/subscription.ts:216]: add a ? to Subscription here. or we need to require the Subscription model above in the query.

Comment (jnaviask) [reply server/models/subscription.ts:216]: It should always be present, so I can require it as a formality, but shouldn't change the result.

Comment (zakhap) [reply server/models/subscription.ts]: I worry about Subscriptions that were deleted but notifications that were improperly deleted from the cascade, but I guess that's not really the kind of issue we should be worried about here.

File: server/models/subscription.ts [hunk 2]
```diff
@@ -189,14 +189,21 @@ export default (
...
-        notification_id: notification.id,
-        subscription_id: subscription.id,
-        is_read: false
-      });
+    const nReads = await models.NotificationsRead.bulkCreate(subscribers.map((subscription) => ({
+      subscription_id: subscription.id,
+      notification_id: notification.id,
+      is_read: false
+    })), {
+      include: {
```
Comment (zakhap) [inline server/models/subscription.ts:197]: After thorough testing: Subscription isn't being returned here with the newly created nRead instance. Not sure how you want to fetch (bulk) the subscriptions to append to the nReads, but the emails below are never sent bc nRead.Subscription is always undefined rn

**Review by zakhap — CHANGES_REQUESTED** [review]

**Review by zakhap — APPROVED** [review]
LGTM! Finally works 🚀  🎸

**Pull request merged**